### PR TITLE
Fixed Snackbar title on toggle of extensions

### LINKED ISSUE #10514: [UI] Better snackbar title on toggle of extensions
### Current Behavior
On toggle for extensions, it says Adapter processing but doesn't show success or failure for enabled extension

### Desired Behavior
Suggest better snackbar title

- Browser: Chrome Safari Firefox
- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by NAMANIND**
**Notes for Reviewers**

This PR fixes #10514 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
Hey @NAMANIND , let's discuss this on today's Meshery Dev meeting at 7:30 PM IST. I am adding this as an [agenda item](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#). Please Join https://meet.layer5.io/meshery if you are available during the meet.

**Comment by NAMANIND** [comment]
@vishalvivekm sure, I will join the meeting!

**Review by sudhanshutech — COMMENTED** [review]
any screenshot or recording @NAMANIND ?

**Comment by NAMANIND** [comment]
Hey @sudhanshutech here is the video



---

## Previously
When enabling or disabling an extension, the message "Adapter processing" was displayed as a popup.

## Currently
Upon enabling or disabling an extension, the name of the extension is properly displayed in the popup message, along with the appropriate "enable" or "disable" text.

---



https://github.com/meshery/meshery/assets/88327184/324b330b-0b7a-4f41-b63b-f77a3d8d24ce

File: ui/components/extensions/adapters/adapters.js [hunk 1]
```diff
@@ -58,19 +58,22 @@ const Adapters = ({ updateProgress, classes }) => {
   const handleAdapterDeployment = (payload, msg, selectedAdapter, adapterId) => {
     updateProgress({ showProgress: true });
 
+    const isEnableAction = payload.status === 'ENABLED';
```
Comment (MUzairS15) [inline ui/components/extensions/adapters/adapters.js:61]: Define (if it does not exist already) a constant inside constants.js and use that here as well on line 74 (you would have to use toLowerCase() or toUpperCase() depending on what you define the value for that constant)

**Review by MUzairS15 — COMMENTED** [review]
A screenshot/recording depicitng the behaviour will be appreciated.

**Comment by MUzairS15** [comment]
More precisely, `<name> adapter enabled/disabled`.

**Comment by sudhanshutech** [comment]
@NAMANIND any further progress?

**Comment by leecalcote** [comment]
Pull request abandoned by author.

@sudhanshutech will you modify as needed based on the feedback and merge?

**Comment by sudhanshutech** [comment]
i have moved and add these and mine changes in another pr: https://github.com/meshery/meshery/pull/11396, so we can close this.

**Pull request closed without merging**